### ISSUE TITLE
Bug: doesn't seem to work with Next.js codemirror multiple instances issue

### ISSUE THREAD

**Issue by 0xdhrv** [codesandbox/sandpack#135]
# Bug report

## Packages affected

- [ ] sandpack-client
- [x] sandpack-react

## Description of the problem
tried to work implement the sandpack-react in next.js

## What were you doing when the problem occurred?
occurred when I tried to work implement the sandpack-react in next.js

### What steps can we take to reproduce the problem?
A simple next.js page:

```js
import { Sandpack } from '@codesandbox/sandpack-react';
import '@codesandbox/sandpack-react/dist/index.css';

export default function ReactPlay() {
  return (
   <Sandpack template="react" />
  );
}
```

![image](https://user-images.githubusercontent.com/19857414/141161802-ebef7bb6-be80-4ece-ac46-88a443e8273b.png)

### Your Environment

| Software                | Name/Version |
| ----------------------- | ------------ |
| Sandpack-client version |              0.2.0|
| Sandpack-react version  |              0.2.1|
| Browser                 |              Google Chrome (Version 95.0.4638.69) |
| Operating System        |              Windows 11|

**Comment by danilowoz** [comment]
Hey @0xdhrv sorry to hear you're facing this problem, we've been facing some problems related to the codemirror dependencies, and I haven't reached a definitive solution. Meanwhile, I'd recommend using the `v0.1.9` version.

**Comment by danilowoz** [comment]
It has been fixed on `v0.3.2`
